pages/enterprise/setup.rst

```diff
@@ -173,3 +173,13 @@ The license automatically applies to all nodes in your cluster without the need
           The same license is also attached as a text file in case it is wrongly formatted in the email.
 
 .. image:: /images/enterprise-license-1.png
+
+
+License Verification
+====================
+
+Some Graylog licenses requires to check their validation on a regular base. This includes the free Enterprise license with a specific amount of traffic included. 
+
+If your network environment requires Graylog to use a proxy server in order to communicate with the external services via HTTPS, you'll have to configure the proxy server in the the :ref:`configuration file of Graylog <http_config>`. 
+
+The Graylog web interface will show you details of the license, but if you are still unclear about the requirements, please contact sales with your questions.
```

Comment:
* s/will show you details of the license/shows all details about the license/
* s/contact sales/contact our sales team/
* Add link to the sales contact form: https://www.graylog.org/contact-sales